WEB/Src/PerformanceCollector/PerformanceCollector/QuickPulseTelemetryModule.cs

```diff
@@ -277,6 +277,23 @@ private static string GetInstanceName(TelemetryConfiguration configuration)
             return string.IsNullOrWhiteSpace(fakeItem.Context?.Cloud?.RoleInstance) ? Environment.MachineName : fakeItem.Context.Cloud.RoleInstance;
         }
 
+        private static string GetRoleName(TelemetryConfiguration configuration)
```

Comment:
Let's not have a separate GetRoleName method doing exactly the same as GetInstanceName that we already have. Let's have a single method that fetches both values, call it only once, and then use its outputs to assign to both instance and roleName.